src/transaction/transaction.rs

```diff
@@ -316,7 +368,9 @@ impl Transaction {
         .await
     }
 
-    /// Pessimistically lock the keys
+    /// Pessimistically lock the keys.
+    ///
+    /// Once resovled it acquires a mutex on the key in TiKV.
```

Comment:
I think it is not really a mutex, just a logical lock

Reply:
Logically it's a mutually exclusive lock 😂. But yes it might be confusing.